client/tls_test.go

```diff
@@ -64,8 +70,13 @@ func TestTLSDefaultsWithCertificate(t *testing.T) {
 
 	numberOfEventsToRead := 1
 	numberOfEvents := uint64(numberOfEventsToRead)
-	_, err = c.ReadAllEvents(context.Background(), direction.Backwards, stream_position.Start{}, numberOfEvents, true)
-	require.NoError(t, err)
+	opts := client.ReadAllOptions{
+		From:           types.Start{},
+		Direction:      types.Backwards,
+		ResolveLinkTos: true,
+	}
+	_, err = c.ReadAll(context.Background(), opts, numberOfEvents)
+	require.True(t, errors.Is(err, io.EOF))
```

Comment:
you actually have `require.ErrorIs`, it will provide nicer output in case of failed assert :)